include/deal2lkit/parsed_solver.h

```diff
@@ -182,7 +182,7 @@ void ParsedSolver<VECTOR>::declare_parameters(ParameterHandler &prm)
 
   ReductionControl::declare_parameters(prm);
 
-  prm.set("Max steps", max_iterations);
+  prm.set("Max steps", std::to_string(max_iterations));
```

Comment:
cool. Another working option is 

prm.set("Max steps", (long int) max_iterations);

I don't know which is the best or if they are equivalent